<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIPanel</class>
 <widget class="QWidget" name="GUIPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIPanel</string>
  </property>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>109</y>
     <width>64</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/EngineButtonStart.png</normaloff>:/img/EngineButtonStart.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pingButton">
   <property name="geometry">
    <rect>
     <x>623</x>
     <y>65</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ping</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>337</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="statusButton">
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Radio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="maximumSize">
     <size>
      <width>271</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string> Detenido</string>
    </property>
   </widget>
  </widget>
  <widget class="QwtDial" name="speedometer">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>22</y>
     <width>117</width>
     <height>118</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="upperBound">
    <double>240.000000000000000</double>
   </property>
   <property name="totalSteps">
    <UInt>240</UInt>
   </property>
   <property name="singleSteps">
    <UInt>1</UInt>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="frameShadow">
    <enum>QwtDial::Sunken</enum>
   </property>
   <property name="origin">
    <double>135.000000000000000</double>
   </property>
   <property name="maxScaleArc">
    <double>270.000000000000000</double>
   </property>
  </widget>
  <widget class="QwtAnalogClock" name="d_clock">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="mode">
    <enum>QwtDial::RotateNeedle</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>104</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Km/h</string>
   </property>
  </widget>
  <widget class="QwtCompass" name="YawCompass">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>187</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="frameShadow">
    <enum>QwtDial::Sunken</enum>
   </property>
  </widget>
  <widget class="QwtSlider" name="speedSlider">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="upperBound">
    <double>240.000000000000000</double>
   </property>
   <property name="scaleMaxMajor">
    <number>6</number>
   </property>
   <property name="scaleMaxMinor">
    <number>3</number>
   </property>
   <property name="scaleStepSize">
    <double>60.000000000000000</double>
   </property>
   <property name="totalSteps">
    <UInt>120</UInt>
   </property>
   <property name="singleSteps">
    <UInt>2</UInt>
   </property>
   <property name="scalePosition">
    <enum>QwtSlider::LeadingScale</enum>
   </property>
   <property name="trough">
    <bool>false</bool>
   </property>
   <property name="groove">
    <bool>true</bool>
   </property>
   <property name="handleSize">
    <size>
     <width>30</width>
     <height>5</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Thrust">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Thrust</string>
   </property>
  </widget>
  <widget class="QwtThermo" name="Fuel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>61</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="alarmLevel">
    <double>20.000000000000000</double>
   </property>
   <property name="maxValue" stdset="0">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="FuelLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fuel (L)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="introFuel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>230</y>
     <width>80</width>
     <height>84</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/img/planeTop.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="headingLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HEADING</string>
   </property>
  </widget>
  <widget class="QwtCompass" name="RollCompass">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>184</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="RollLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ROLL</string>
   </property>
  </widget>
  <widget class="QLabel" name="NoPitchLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>No Pitch</string>
   </property>
  </widget>
  <widget class="QwtCompass" name="PitchCompass">
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>185</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>253</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/img/YAW.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="RollLabel_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PITCH</string>
   </property>
  </widget>
  <widget class="QLabel" name="pitchLabel">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>257</y>
     <width>85</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/img/PITCH.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Height">
   <property name="geometry">
    <rect>
     <x>486</x>
     <y>29</y>
     <width>46</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QLabel" name="meter">
   <property name="geometry">
    <rect>
     <x>493</x>
     <y>104</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(meter)</string>
   </property>
  </widget>
  <widget class="QLabel" name="serialPortLabel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Puerto Serie:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="serialPortComboBox">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>35</y>
     <width>96</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Counter" name="height">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>55</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="brokenGlass">
   <property name="geometry">
    <rect>
     <x>-104</x>
     <y>0</y>
     <width>808</width>
     <height>534</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>808</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/img/broken-glass.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>34</y>
     <width>28</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/gtk_refresh.png</normaloff>:/img/gtk_refresh.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="AutoPilot">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>155</y>
     <width>102</width>
     <height>47</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>AUTOPILOT ENGAGED</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Km/h</string>
   </property>
  </widget>
  <zorder>brokenGlass</zorder>
  <zorder>serialPortLabel</zorder>
  <zorder>serialPortComboBox</zorder>
  <zorder>YawCompass</zorder>
  <zorder>runButton</zorder>
  <zorder>pingButton</zorder>
  <zorder>splitter_2</zorder>
  <zorder>speedometer</zorder>
  <zorder>d_clock</zorder>
  <zorder>label</zorder>
  <zorder>speedSlider</zorder>
  <zorder>Thrust</zorder>
  <zorder>Fuel</zorder>
  <zorder>FuelLabel</zorder>
  <zorder>introFuel</zorder>
  <zorder>label_2</zorder>
  <zorder>headingLabel</zorder>
  <zorder>RollCompass</zorder>
  <zorder>RollLabel</zorder>
  <zorder>NoPitchLabel</zorder>
  <zorder>PitchCompass</zorder>
  <zorder>label_3</zorder>
  <zorder>RollLabel_2</zorder>
  <zorder>pitchLabel</zorder>
  <zorder>Height</zorder>
  <zorder>meter</zorder>
  <zorder>height</zorder>
  <zorder>refreshButton</zorder>
  <zorder>AutoPilot</zorder>
  <zorder>label_4</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Counter</class>
   <extends>QWidget</extends>
   <header>analogwidgets/counter.h</header>
  </customwidget>
  <customwidget>
   <class>QwtAnalogClock</class>
   <extends>QwtDial</extends>
   <header>qwt_analog_clock.h</header>
  </customwidget>
  <customwidget>
   <class>QwtCompass</class>
   <extends>QwtDial</extends>
   <header>qwt_compass.h</header>
  </customwidget>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
  <customwidget>
   <class>QwtSlider</class>
   <extends>QWidget</extends>
   <header>qwt_slider.h</header>
  </customwidget>
  <customwidget>
   <class>QwtThermo</class>
   <extends>QWidget</extends>
   <header>qwt_thermo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>serialPortComboBox</tabstop>
  <tabstop>pingButton</tabstop>
  <tabstop>runButton</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
